docs/userguide/basics.visualisation.rst

```diff
@@ -114,3 +114,31 @@ To use a different scene object as the parent, the parent attribute of the scene
 >>> boxobj = scene.add(box, parent=pointobj)
 >>> boxobj.parent
 PointObject
+
+
+Scene Context
+=============
+
+Depends on where the code is executed, the "scene" will detect the current visualisation context.
+If the code is executed in the Rhino, the scene context will be automatically set as "Rhino". 
+
+>>> from compas.scene import Scene
+>>> from compas.geometry import Box
+>>> scene = Scene()
+>>> scene.context
+Rhino
+
+Depends on the context, the different implementations of the scene objects will be used.
```

Comment:
For every context, the appropriate scene object implementations will be used automatically.